packages/block-library/src/query-title/edit.js

```diff
@@ -25,9 +25,10 @@ import { __, _x, sprintf } from '@wordpress/i18n';
  * Internal dependencies
  */
 import { useArchiveLabel } from './use-archive-label';
+import { usePostTypeLabel } from './use-post-type-label';
 import { useToolsPanelDropdownMenuProps } from '../utils/hooks';
 
-const SUPPORTED_TYPES = [ 'archive', 'search' ];
+const SUPPORTED_TYPES = [ 'archive', 'search', 'post-type-label' ];
```

Comment:
```suggestion
const SUPPORTED_TYPES = [ 'archive', 'search', 'post-type' ];
```

`-label` may not be necessary for consistency with other types.